.cirrus.yml

```diff
@@ -987,6 +987,22 @@ test_image_build_task:
     env:
         DRYRUN: 1
 
+fcos_image_build_task:
+    # Only a test. Does not push built images.
+    # Image push to quay will be handled by GHA in
+    # .github/workflows/fcos-podman-next-build.yml
+    <<: *image-build
+    name: "Test Build FCOS image $CTXDIR"
```

Comment:
Hrmmm, sorry, just looked.  I don't see this Cirrus-task listed as running, I was expecting it should have, no?  Maybe I gave bad advice on the `only_if` condition?

Reply:
I found in one of the job logs, `CIRRUS_PR=19477` and `CIRRUS_BRANCH=pull/19477`.  Hrmmmm :thinking:

Reply:
Ahh, try `$CIRRUS_BASE_BRANCH`.  That's the one to use in a PR context....so confusing :confused:

Reply:
This still needs fixing, the `only_if` condition is wrong - `$CIRRUS_BRANCH` won't work, need to use `$CIRRUS_BASE_BRANCH`.

Reply:
did you check the latest ? I had some rebase issues so you might have seen an older version.